IFC4x3/PropertyConstants/p/PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PARALLEL_WITH_CORRECTIVE_RINGS_3CMdaJ0BDCQg7JquajrJyX" Name="PARALLEL_WITH_CORRECTIVE_RINGS" UniqueId="cc5a7913-00b3-4c6a-a1d3-d3892dd53f21" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)